extensions/wikia/AdEngine/src/templates/porvata-config.js

```diff
@@ -0,0 +1,21 @@
+import slots from '../slots';
+
+function getNavbarHeight() {
+	const navbar = document.getElementById('globalNavigation');
+
+	return navbar ? navbar.offsetHeight : 0;
+}
+
+export const getConfig = () => ({
+		inViewportOffsetTop: getNavbarHeight(),
+		isFloatingEnabled: true,
+		onInit: (adSlot, params) => {
+			params.isVideoMegaEnabled = true;
```

Comment:
Does it work with recent MEGA changes? I mean outstream ad unit which has {_dbname} instead of "top_1k"

Reply:
Solved :+1: